swarm/storage/ldbstore.go

```diff
@@ -290,8 +320,8 @@ func (s *LDBStore) collectGarbage(ratio float32) {
 
 	sort.Slice(garbage[:gcnt], func(i, j int) bool { return garbage[i].value < garbage[j].value })
 
-	cutoff := int(float32(gcnt) * ratio)
-	metrics.GetOrRegisterCounter("ldbstore.collectgarbage.delete", nil).Inc(int64(cutoff))
+	cutoff := int(float32(gcnt)) //* ratio)
+	metrics.GetOrRegisterCounter("ldbstore.collectgarbage.delete", nil).Inc(int64(maxGcnt))
```

Comment:
if we are going to remove `cutoff` elements, then this counter needs to be incremented with `cutoff`, not with `maxGnct`